src/modules/did.js

```diff
@@ -49,19 +48,20 @@ class ChluIPFSDID {
     }
 
     isPresent() {
-        return this.publicDidDocument && this.didId && this.privateKeyBase58
+        return Boolean(this.publicDidDocument && this.didId && this.privateKeyBase58)
```

Comment:
We could use lodash here again. I don't know why, but I am still afraid of implicit testing of null/undefined/empty variables in js.